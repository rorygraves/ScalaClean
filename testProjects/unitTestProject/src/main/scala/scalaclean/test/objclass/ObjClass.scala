

object Foo {
  def myMethod(): Int = 1
}

class Foo {
  def myMethod(): Int = 2
}