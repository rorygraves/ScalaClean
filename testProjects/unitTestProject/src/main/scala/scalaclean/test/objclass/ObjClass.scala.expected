

object Foo {
  def myMethod(): Int = 1/* allOutgoingReferences(O:<empty>.Foo/M:myMethod()) - C:scala.Int :: P:scala */
}/* allOutgoingReferences(O:<empty>.Foo) - <root> :: C:java.lang.Object :: C:java.lang.Object/M:<init>() :: C:scala.Unit :: O:<empty>.Foo/V:<local Foo> :: P:<empty> :: P:java.lang :: P:scala :: P:scalaT:AnyRef */

class Foo {
  def myMethod(): Int = 2/* allOutgoingReferences(C:<empty>.Foo/M:myMethod()) - C:scala.Int :: P:scala */
}/* allOutgoingReferences(C:<empty>.Foo) - <root> :: C:<empty>.Foo/V:<local Foo> :: C:java.lang.Object :: P:<empty> :: P:scala :: P:scalaT:AnyRef */
